### DISCUSSION TITLE
Simple Japanese typing game

### DISCUSSION THREAD

**Discussion by swkidd** [KilledByAPixel/LittleJS#85] [Show and tell]
I created a simple typing game to learn Japanese words! The goal is to type the words before they hit the ground. Unknown words will show on the screen so they can be learned. I couldn't find a nice way to show the keyboard on iOS so it is currently only working on desktop.

code: https://github.com/langgames/langgames.github.io

[Click to play](https://lang.games)

**Comment by langgames** [comment]
I've created a keyboard and gamepad component here:

keyboard: https://github.com/langgames/langgames.github.io/blob/main/lib/keyboard.js
gamepad: https://github.com/langgames/langgames.github.io/blob/main/lib/gamepad.js

**Comment by KilledByAPixel** [comment]
Very cool, I wish I knew Japanese so I could play!

Keep posting updates for us, it's great seeing your progress.
